### DISCUSSION TITLE
Custom page in Strapi AdminPanel

### DISCUSSION THREAD

**Discussion by iamwebdev** [strapi/strapi#7812] [Legacy Topics]
OS: linux
strapi: 3.1.0
db: mongo
node:12

I want to create a new custom page in admin panel of strapi with dynamic data

**Comment by derrickmehaffy** [comment]
Hello!

The best option in your case would be to built a plugin, you can find the quickstart here: https://strapi.io/documentation/v3.x/plugin-development/quick-start.html#quick-start

And on the left there are various pages about backend and frontend development for the plugin along with the APIs.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
